boards/arm/nrf52810_pca20045/nrf52810_pca20045.yaml

```diff
@@ -0,0 +1,16 @@
+identifier: nrf52810_pca20045
+name: nRF52810-PCA20045
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
+supported:
+  - adc
+  - ble
+  - pwm
+  - watchdog
+  - counter
```

Comment:
Are all these "supported" entries needed for this board? I'm not sure what is their effect here downstream, but upstream they cause that CI builds drivers tests for them and some of these tests (at least the ones for `adc` and `counter`) require additional configurations. Maybe it would be safer to not add these entries? Unless they are really required for something.

Reply:
I don't need them for sure just followed the upstream. If you think they are not needed I will remove them.